conf/turnkey.d/cronapt

```diff
@@ -15,37 +15,41 @@ EOF
 
 echo -n > action.d/3-download
 
-cat > action.d/5-install.README << EOF
-# These files named '5-install.XXX' relate to the '5-install' cronapt script
-# located in /etc/cron-apt/actions.d/
+cat > TurnKey_Linux_5-install.README << EOF
+# The files in actiona-available.d named '5-install.XXX' relate to the
```

Comment:
Is `actiona-available.d` a typo (did you intend `actions-available.d` or `action-available.d` instead)?

Reply:
Opps, yep typo in the readme!

IO'll fix that and merge! :smile: